### PR TITLE
Add rule for python-ldap 3.2.0

### PR DESCRIPTION
**Pull request by natefoo**

### CODE REVIEW

File: wheels/python-ldap/meta.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+name: python-ldap
+version: 3.2.0
+type: wheel
+purepy: false
+build: 1
+src:
+  - https://gmplib.org/download/gmp/gmp-6.1.2.tar.bz2
+  - https://ftp.gnu.org/gnu/nettle/nettle-3.4.1.tar.gz
+  - https://www.gnupg.org/ftp/gcrypt/gnutls/v3.6/gnutls-3.6.7.tar.xz
+  - http://www.openssl.org/source/openssl-1.0.2r.tar.gz
```
Comment (nsoranzo) [inline wheels/python-ldap/meta.yml]:
```suggestion
  - https://www.openssl.org/source/openssl-1.0.2r.tar.gz
```

File: wheels/python-ldap/meta.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  - http://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.4.47.tgz
```
Comment (nsoranzo) [inline wheels/python-ldap/meta.yml]:
```suggestion
  - https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.4.47.tgz
```

File: wheels/python-ldap/meta.yml (hunk 1 above)
Comment (nsoranzo) [reply wheels/python-ldap/meta.yml]: Also, the OpenSSL 1.0.2 series will be supported until 2019-12-31, should we try with 1.1.1b ?

**Comment by natefoo** [comment]
OpenSSL 1.1.1's build process requires Perl 5.10, which isn't available in the manylinux1 image. I'm going to switch it back to 1.0.2 and let's shoot for building manylinux2010 wheels before the end of the year.

**Review by nsoranzo — APPROVED** [review]

**Comment by nsoranzo** [comment]
Thanks @natefoo ! Should we add an issue to the starforge repo about manylinux2010 ?

**Comment by natefoo** [comment]
Sure!

**Pull request merged**